### ISSUE TITLE
[cc] constexpr breaks highlighting and indent

### ISSUE THREAD

**Issue by mtaran-google** [codemirror/google-modes#87]
```cpp
constexpr char foo[] = "blah";
constexpr char bar[] = "blah";
```
problems: `bar` isn't highlighted, and if you press return after the first semicolon, there is an unexpected indent

**Comment by marijnh** [comment]
Could it be that this was solved by 161b6acc7c2d3a? I can't reproduce it with the current code.

**Comment by mtaran-google** [comment]
Yes, I can't repro it now either. Thanks.
